src/redux/categories/categories.js

```diff
@@ -0,0 +1,18 @@
+const CHECK_STATUS = 'category/CHECK_STATUS';
+
```

Comment:
- [ ] Kindly create actions using the _**duck pattern**_. For example, `const CHECK_STATUS = 'bookstore/category/CHECK_STATUS'`. You can read this [article](https://github.com/erikras/ducks-modular-redux) for more information. 👍

Reply:
Will do 👍